plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/GraphQLPluginExtension.kt

```diff
@@ -59,6 +59,8 @@ open class GraphQLPluginClientExtension {
     var endpoint: String? = null
     /** GraphQL server SDL endpoint that will be used to download schema. Alternatively you can run introspection query against [endpoint]. */
     var sdlEndpoint: String? = null
+    /** GraphQL schema file location. Can be used instead of [endpoint] or [sdlEndpoint]. */
+    var schemaFile: File? = null
```

Comment:
Do we want to use a file here or load the file for users and just accept a string? @dariuszkuc any comments?

Reply:
Yes, this should be `var schemaFileName: String? =null` (matching https://github.com/ExpediaGroup/graphql-kotlin/blob/master/plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/tasks/AbstractGenerateClientTask.kt#L60). Lets also update javadocs for the `endpoint`/`sdlEndpoint` to mention that you can specify file directly as well.